src/static/manifest.json

```diff
@@ -15,7 +15,8 @@
     "default_popup": "popup.html"
   },
   "background": {
-    "service_worker": "js/backgroundScript.js"
+    "service_worker": "js/backgroundScript.js",
+    "scripts": ["js/backgroundScript.js"]
```

Comment:
Does this work with chrome?

Reply:
Will chrome also make a stink if this is there?

Reply:
It works on chrome as well.